src/modules/todolist.js

```diff
@@ -71,5 +71,3 @@ class ToDoList {
       this.updateTasks();
```

Comment:
- [ ] Great work implementing the  status updates method, but you should create a new JavaScript file imported as a module, please move your code to a new file

Reply:
@Ismailco please see project again . They are already imported as a module .

Reply:
You need a new file

Reply:
@ismailco 
just a newfile.js for you Sir with every lines of code from the same todolist?  even the project works everything well ?
https://silviatofana.github.io/todolist/dist/